modules/websocket_bridge/include/hephaestus/ipc/ipc_graph.h

```diff
@@ -0,0 +1,158 @@
+//=================================================================================================
```

Comment:
QQ: this file is going to be moved to the `ipc` module, right?

Reply:
I was preparing it for such a move, yes. Will give it a quick try, otherwise it has to be in a follow up PR.

Reply:
Done